applications/openshift/authentication/oauth_or_oauthclient_inactivity_timeout/kubernetes/shared.yml

```diff
@@ -0,0 +1,8 @@
+# platform = multi_platform_ocp
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  tokenConfig:
+    accessTokenInactivityTimeout: 600s
```

Comment:
Even if the kubernetes remediations can't be configured, I thought it was better to have an automated remediation with some reasonable default. Let me know if you think this would cause more problems for the customers and I can remove it

Reply:
Nope, I think this is a good approach. Can the Compliance Operator already remediate this? didn't know we had already set the RBAC permissions for this.

Reply:
Oh you're right. When I was looking at what RBAC perms we need for the fedramp work, I did add read perms for the OAuth objects, but forgot that we also need write perms for the remediations.

I suggest we split the PR into two, merge the check without remediation and when a new CO release is ready (I guess we might find more RBAC issues), then add the remediation.